apps/wallet/src/ui/app/components/active-coins-card/CoinItem.tsx

```diff
@@ -29,19 +29,23 @@ export function CoinItem({
 		<div className="flex gap-2.5 w-full py-3 pl-1.5 pr-2 justify-center items-center rounded hover:bg-sui/10">
 			<CoinIcon coinType={coinType} size={isActive ? 'sm' : 'md'} />
 			<div className="flex flex-1 gap-1.5 justify-between items-center">
-				<div className="flex flex-col gap-1.5">
+				<div className="block max-w-token-width">
```

Comment:
is block required here? block is the default value right?

Reply:
remove block usage